charts/osm/values.schema.json

```diff
@@ -701,6 +701,16 @@
                         "error"
                     ]
                 },
+                "localProxyMode": {
+                    "$id": "#/properties/osm/properties/localProxyMode",
+                    "type": "string",
+                    "title": "The localProxyMode schema",
+                    "description": "Proxy mode for the Envoy proxy sidecar. Acceptable values are ['Localhost', 'PodIP'].",
+                    "pattern": "^(Localhost|PodIP)$",
```

Comment:
Can this be an enum?

Reply:
We didn't use it anywhere else in the helm schema so I didn't know it was a thing; added 👍🏾